code-studio/src/js/initializeCodeMirror.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file Function that initializes a CodeMirror editor in place of a textarea.
+ */
+/* global inlineAttach */
+'use strict';
```

Comment:
There appears to be a [plugin for enforcing strict mode across a whole bundle](https://www.npmjs.com/package/strictify) but we're not using it.  Since browserify wraps each file as a module, we're technically using [function-level strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Strict_mode_for_functions) here.  Personally, I prefer using strict mode as much as possible because it catches a few common mistakes (and JSHint adjusts accordingly).